studies/index.md

```diff
@@ -70,6 +70,67 @@ devices) on the studies page.
    make sure the machine has access to your repository. Once a branch is
    selected click `Link study`.
 
+### uploading experiments
+
+To upload new experiments all that is needed is to push experiments to a
+specific branch of the git repository. For instance if a study was created with
+an experiment branch called `experiments-1` the workflow would be as followed:
+
+```shell
+# Clone the repository
+# you can find the url on
+# https://exporl.med.kuleuven.be/apex-experiments/
+# under projects.
+git clone <url>
+cd <repository>
+# First create a commit.
+git add <experiment files and stimuli>
+git commit
+
+# Push to the experiment branch.
+git push origin HEAD:refs/heads/experiments-1
+# If you want to do a gerrit review first, push to refs/for.
+git push origin HEAD:refs/for/experiments-1
+```
+
+### fetching results
+
+Fetching the results is somewhat complex because the result branches don't have
+any shared history.
+
+```shell
+# Clone the repository
+# you can find the url on
+# https://exporl.med.kuleuven.be/apex-experiments/
+# under projects.
+git clone <url>
+cd <repository>
+
+# Checkout a branch to collect the results on.
+git checkout --orphan collected-results
+git rm --cached -r .
+git clean -fd
+
+# Create a base commit, merging won't work without one.
+git commit --allow-empty -m "Base commit for merge."
+
+# List all result branches.
+BRANCHES=$(git branch -a --list origin/results-*)
+# Merge all branches onto collected-results.
+# The actual merge will fail,
+# but it will give the right history.
+git merge --allow-unrelated-histories -s octopus $BRANCHES
+# Read the content of each branches into the index.
+git read-tree $BRANCHES
+# Commit them into collected-results.
+git commit -m "Merge $BRANCHES into collected-results."
+# Reset index.
+git reset --hard
+```
+
+The results should now be collected on the `collected-results` branch, and
+present in your working directory.
+
```

Comment:
In a typical workflow, you'd want to do this a number of times when new results come in. What would be the best strategy to deal with that? Start over? Or merge into the existing one?